app/src/main/java/com/cornellappdev/uplift/ui/screens/MainReminderScreen.kt

```diff
@@ -0,0 +1,113 @@
+package com.cornellappdev.uplift.ui.screens
+
+import androidx.compose.animation.Crossfade
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavHostController
+import com.cornellappdev.uplift.models.LocationRepository
+import com.cornellappdev.uplift.networking.ApiResponse
+import com.cornellappdev.uplift.ui.screens.subscreens.MainError
+import com.cornellappdev.uplift.ui.screens.subscreens.MainLoaded
+import com.cornellappdev.uplift.ui.screens.subscreens.MainLoading
+import com.cornellappdev.uplift.ui.viewmodels.ClassDetailViewModel
+import com.cornellappdev.uplift.ui.viewmodels.GymDetailViewModel
+import com.cornellappdev.uplift.ui.viewmodels.HomeViewModel
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.rememberMultiplePermissionsState
+import com.valentinilk.shimmer.Shimmer
+import kotlinx.coroutines.delay
+import android.util.Log
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.Icon
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.TopAppBarColors
+import androidx.compose.material3.TopAppBarDefaults
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.cornellappdev.uplift.R
+import com.cornellappdev.uplift.util.GRAY00
+import com.cornellappdev.uplift.util.GRAY01
+import com.cornellappdev.uplift.util.LIGHT_GRAY
+import com.cornellappdev.uplift.util.montserratFamily
+import com.google.accompanist.systemuicontroller.SystemUiController
+import com.google.accompanist.systemuicontroller.rememberSystemUiController
+
+/**
+ * The home page of Uplift.
+ */
+@OptIn(ExperimentalPermissionsApi::class, ExperimentalMaterial3Api::class)
```

Comment:
Android Studio is telling me that the ExperimentalPermissionsApi is not used so the marker is probably unnecessary